src/app/user/login/post.controller.js

```diff
@@ -38,7 +38,18 @@ module.exports = (req, res) => {
           if (Object.prototype.hasOwnProperty.call(user, 'message')) {
             logger.info(`Invalid email entered: ${usrname}`);
             cookie.setUserVerified(false);
-            res.redirect('/login/authenticate');
+            emailService
```

Comment:
Probably worth checking to see if the message == user not registered
With the above an unverified user will get the email which shouldn't happen